x/cdp/keeper/querier.go

```diff
@@ -17,6 +18,8 @@ func NewQuerier(keeper Keeper) sdk.Querier {
 		switch path[0] {
 		case types.QueryGetCdp:
 			return queryGetCdp(ctx, req, keeper)
+		case types.QueryGetV2Cdps:
+			return queryGetV2Cdps(ctx, req, keeper)
 		case types.QueryGetCdps:
```

Comment:
are you planning on deprecating this query?

Reply:
Ongoing discussion at https://github.com/Kava-Labs/kava/pull/644#discussion_r481130316. The original idea was to maintain existing endpoints as to limit breaking changes for kava-4, but that's more relevant on the REST API as it's the primary point of data consumption.

Reply:
Removed in https://github.com/Kava-Labs/kava/pull/644/commits/6f8355c26f39aa6e0908214bd67621e02cd0f760